sdk/cognitivelanguage/ai-language-text/samples-dev/healthcare.ts

```diff
@@ -85,6 +89,9 @@ export async function main() {
             console.log(`\t\t\t- "${role.entity.text}" with the role ${role.name}`);
           }
         }
+        if (result.fhirBundle) {
```

Comment:
can't comment on the line, but should we show the new confidenceScore for relations in the sample?

Reply:
Samples should show champion scenarios and I am not sure if using confidence scores in this case is one. I also didn't show the confidence scores for entities. Let me know if you feel strongly about this.